examples/strong/analyzer-results-dev.txt

```diff
@@ -1,18 +1,11 @@
 Analyzing analysis_options.yaml, lib, test...
-  error • 'HoneyBadger.parent' ('Root Function()') isn't a valid override of 'Animal.parent' ('Animal Function()'). • lib/animal_bad.dart:15:12 • invalid_override
```

Comment:
Is this file different from analyzer-results-stable.txt? If not, then maybe we should remove one of these files and delete "-stable" from the name.

Reply:
It's identical.